data/data/aws/route53/variables.tf

```diff
@@ -8,19 +8,19 @@ variable "base_domain" {
   type        = "string"
 }
 
-variable "master_count" {
-  description = "The number of masters"
+variable "controlplane_count" {
+  description = "The number of control plane nodes"
```

Comment:
nit: I usually hyphenate the adjectival form ("control-plane nodes"), but I dunno what outstanding precedent there is for either approach (I expect there are lots examples of both ways ;).